packages/story-editor/src/components/popup/index.js

```diff
@@ -99,10 +99,11 @@ function Popup({
       if (evt?.target?.nodeType && popup.current?.contains(evt.target)) {
         return;
       }
+
       setPopupState({
-        offset:
-          anchor?.current &&
-          getOffset(placement, spacing, anchor, dock, popup, isRTL, topOffset),
+        offset: anchor?.current
+          ? getOffset(placement, spacing, anchor, dock, popup, isRTL, topOffset)
+          : {},
```

Comment:
https://github.com/GoogleForCreators/web-stories-wp/blob/f9530da9116b559494381dbc28abda834ea0dce8/packages/story-editor/src/components/popup/index.js#L92-L110

If `anchor` is `undefined` - should this function even set the popup state?

Reply:
Yes I think we do want it to set back to an empty object to clear the popupState.

Reply:
Looks like the same is used in the popup of the `design-system`, maybe change it there as well?

Once #10089 gets merged, only the `design-system` popup will be used.

Reply:
Hm, okay, looks like the PR got merged without this already. Another PR? 😛 Not sure if the `design-system` Popup is causing any crashes currently, I have seen the right-click menu crashing once but haven't been able to repro.

Reply:
Sorry about that, I missed this thread. A new PR would be good, or just push changes to #10197 to bring it over the finish line